StarRailBattleSim/src/lightcones/nihility/PatienceIsAllYouNeed.java

```diff
@@ -0,0 +1,35 @@
+package lightcones.nihility;
+
+import characters.AbstractCharacter;
+import enemies.AbstractEnemy;
+import lightcones.AbstractLightcone;
+import powers.PermPower;
+import powers.PowerStat;
+
+import java.util.ArrayList;
+
+public class PatienceIsAllYouNeed extends AbstractLightcone {
+
+    public PatienceIsAllYouNeed(AbstractCharacter owner) {
+        super(1058, 582, 463, owner);
+    }
+
+    @Override
+    public void onEquip() {
+        this.owner.addPower(PermPower.create(PowerStat.DAMAGE_BONUS, 24, "Patience Is All You Need DMG Boost"));
+    }
+
+    @Override
+    public void onAttack(AbstractCharacter character, ArrayList<AbstractEnemy> enemiesHit, ArrayList<AbstractCharacter.DamageType> types) {
+        if (character == null) return;
+        character.addPower(new PatienceIsAllYouNeedSpeedBoost());
+    }
+
+    public static class PatienceIsAllYouNeedSpeedBoost extends PermPower {
```

Comment:
Stacking speed buffs don't currently have a hook.

Reply:
Added a throw in the constructor